index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Newton Portfolio Project</title>
+  <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+  <main>
+    <nav>
+      <div class="desktop-menu">
+        <ul>
+          <li>
+            <a href="">Portfolio</a>
+          </li>
+          <li>
+            <a href="">About</a>
+          </li>
+          <li>
+            <a href="">Contact</a>
+          </li>
+        </ul>
+      </div>
+      <div class="mobile-menu-icon">
+        <img src="./images/Union.svg" alt="">
+      </div>
+    </nav>
+    <section class="home">
+      <div class="introduction">
+        <h2>Hey there.</h2>
+        <h2>I'm Newton.</h2>
+        <h2>I'm a software developer</h2>
+      </div>
+      <div class="content">
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      </div>
+      <div class="social-icons">
+        <img src="./images/twitter.svg" alt="">
+        <img src="./images/Linkedin icon.svg" alt="">
+        <img src="./images/medium.svg" alt="">
+        <img src="./images/github.svg" alt="">
+        <img src="./images/angellist.svg" alt="">
+      </div>
```

Comment:
For the list of icons, kindly use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. 🤓 🛠